src/sentry/api/endpoints/system_health.py

```diff
@@ -17,9 +17,9 @@
 @all_silo_endpoint
 class SystemHealthEndpoint(Endpoint):
     publish_status = {
-        "GET": ApiPublishStatus.UNKNOWN,
+        "GET": ApiPublishStatus.PRIVATE,
     }
-    owner = ApiOwner.DATA
+    owner = ApiOwner.OPEN_SOURCE
```

Comment:
This looked open source to me. @azaslavsky ?

Reply:
👍